compiler-core/src/language_server/tests/snapshots/gleam_core__language_server__tests__action__destructure_argument_empty_tuple.snap

```diff
@@ -0,0 +1,18 @@
+---
+source: compiler-core/src/language_server/tests/action.rs
+expression: "\npub fn main(tuple: #()) {\n  todo\n}\n"
+---
+----- BEFORE ACTION
+
+pub fn main(tuple: #()) {
+            ↑            
+  todo
+}
+
+
+----- AFTER ACTION
+
+pub fn main(tuple: #()) {
+  let #() = tuple
```

Comment:
I think we shouldn't offer the action when there's no values to match on

Reply:
Ok, fixed!